@xen-orchestra/web-core/lib/components/ui/tag/UiTagsList.vue

```diff
@@ -0,0 +1,14 @@
+<!-- v1 -->
+<template>
+  <div class="ui-tags-list">
+    <slot />
+  </div>
+</template>
+
+<style lang="postcss" scoped>
+.ui-tags-list {
+  display: flex;
+  flex-wrap: wrap;
```

Comment:
Is flex-wrap necessary here ? Won't this potentially break the design? I'm not sure where this is going to be used

Reply:
It is when the div is too narrow, I think I would like the tags to go below.

Reply:
**without wrap**
<img width="1499" alt="Capture d’écran 2024-11-29 à 08 10 30" src="https://github.com/user-attachments/assets/5687ced4-8689-483b-8397-d0b156585d06">

**with wrap**
<img width="1045" alt="Capture d’écran 2024-11-29 à 08 10 42" src="https://github.com/user-attachments/assets/75af2c33-22fe-4f45-a9ca-eb244a334449">

Reply:
yes, that was the question I was asking myself, if you knew of a place where it was used !
Sounds good for me

Reply:
Yes, it is the correct behavior, example in XO Lite from Figma:
![image](https://github.com/user-attachments/assets/4934a14e-62c5-4413-8cef-920d3bcdbcb4)
I didn't find any other examples that would not wrap. We could add an option later if needed.